terraso_backend/apps/shared_data/migrations/0010_visualizationconfig_mapbox_tileset_id_status.py

```diff
@@ -0,0 +1,43 @@
+# Copyright © 2023 Technology Matters
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/.
+
+# Generated by Django 4.2.2 on 2023-06-26 23:21
+
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
```

Comment:
question: Do you know why the migration depends on the user auth model? Glancing over it I can't see any dependency.

Reply:
@david-code that is a good question, I used makemigrations just for the shared_data app, maybe that made this different, I will check it

Reply:
I removed this, it is not needed